x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -402,6 +416,59 @@ export class AlertsClient<
     return this.legacyAlertsClient.factory();
   }
 
+  public async getPersistentAlerts({
```

Comment:
nit: I think `getSummarizedAlerts` is still more descriptive than `getPersistentAlerts`. We persist all alerts as data documents but this function returns a summarized form of the docs that are persisted

Reply:
done